src/etc/inc/dyndns.class

```diff
@@ -702,6 +703,19 @@
 					curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
 					curl_setopt($ch, CURLOPT_URL, $server . '?domains=' . str_replace('.duckdns.org', '', $this->_dnsHost) . '&token=' . $this->_dnsUser);
 					break;
+				case 'googledomains':
+					if ($this->_dnsVerboseLog)
+						log_error("Google Domains: ({$this->_dnsHost}): DNS update() starting.");
+					$server = "https://domains.google.com/nic/update";
+					$post_data['hostname'] = $this->_dnsHost;
+					$post_data['myip'] = $this->_dnsIP;
+					$post_data['offline'] = 'no';
+					curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
```

Comment:
If I look at the certificate, Google uses an own CA as an intermediate CA which is signed by GeoTrust Global CA.

![spectacle db3271](https://cloud.githubusercontent.com/assets/3713914/23096502/567d22f8-f61e-11e6-9759-9130269ab8f0.png)

Maybe curl does not check for the next certificate in the chain if it is missing. In this case it would be better, if the CA is added to the global store or pinned for this curl request.